keps/sig-apps/3973-consider-terminating-pods-deployment/README.md

```diff
@@ -210,12 +213,20 @@ This feature is already implemented for Jobs ([KEP-3939](https://github.com/kube
 
 ### Risks and Mitigations
 
+#### Feature Impact
+
 Deployment rollouts might be slower when using the `TerminationComplete` PodReplacementPolicy.
 
 Deployment rollouts might consume excessive resources when using the `TerminationStarted` PodReplacementPolicy.
 
 This is mitigated by making this feature opt-in.
 
+#### kubectl Skew
+
+The newly added `deployment.kubernetes.io/replicaset-replicas-before-scale` annotation should be
+removed during rollback. Support for this will be added to kubectl in the same release as this feature.
+Therefore, older versions of kubectl are not supported in this feature alpha.
```

Comment:
1. Do we need to update https://kubernetes.io/releases/version-skew-policy/#kubectl to explain or caveat this case?

   (I would).

1. This KEP doesn't just define an alpha; it defines something we intend to one day stabilize. So:
   ```suggestion
   The `deployment.kubernetes.io/replicaset-replicas-before-scale` annotation should be
   removed during rollback. Support for this removal will be added to kubectl in the same release as this feature.
   Therefore, rollback using an older kubectl will not be supported until one minor release after the feature
   first reaches alpha. The documentation for Deployments will include a notice about this.
   ```

However, also see PLACEHOLDER - I would expect the annotation to be managed by the control plane, not by kubectl.

Reply:
@sftim 

I went over the whole logic again and found out that the skew bug is more nuanced and has less impact than I thought before.

I have updated the kubectl Skew section. PTAL, and thank you for your suggestion. I also added a mention to the graduation. Since the impact is not so big and it is still an alpha, I would vote for just mentioning the quirk in the deployment documentation together with the feature

Also added additional steps to the enablement and re-enablement.